cpu/sam0_common/include/vendor/README.md

```diff
@@ -41,6 +41,13 @@ changes will be lost when a new ASF release is going to be used.
 A SAM based CPU should include `sam0.h` in this directory, which will
 resolve  any CPU family specific includes required.
 
+## SAMD10 files
+
+samd10: `Atmel.SAMD10_DFP.1.1.77.atpack`
+
+Each atpack has an include subdirectory with the files we copy into
+RIOT. The files are copied unmodified.
```

Comment:
Any chance to implement this as package instead?

Reply:
Unfortunately I didn't find a github mirror of those.
Microchip only offers [zip files](http://packs.download.atmel.com/) for download that then were manually added to RIOT.

Reply:
The nordic softdevice ble package is also downloaded as zip. Or the c25519 package. So I guess this should work with zip files as well.

Reply:
(alternatively we can also maintain our own HAL repo [like Zephyr](https://github.com/zephyrproject-rtos/hal_atmel) - or just add those files to their repo :smile:)

Reply:
But as this would have to be done for all members of the sam0 family, I relegate that to another PR.